### DISCUSSION TITLE
Running only a single instance of stellarium?

### DISCUSSION THREAD

**Discussion by RobertNufer** [Stellarium/stellarium#3336] [Q&A]
Is it possible to start stellarium in one instance only? (I am using Windows 10)
I have many small scripts saved as .ssc files.
By double-clicking on one of theose files  only the currently running instance of stellarium should be "fed" with the script.
Thank you

**Comment by gzotti** [comment]
No, we often use several instances at once, on purpose. It is not possible to "push" a script to a running instance by calling the program again from the the command line. You can use the RemoteControl API for that.

**Reply by RobertNufer** [reply]
Georg,
thank you very much for guiding me to the RemoteControl API.
I installed curl on my windows PC.
I have written a .cmd file for testing...

**GET**
**curl**  http://localhost:8090/api/scripts/info?id="_jovian.ssc"     can be run from command prompt with
**Start** http://localhost:8090/api/scripts/info?id="_jovian.ssc".
Both versions return the requested information from the script.

**curl**  http://localhost:8090/api/scripts/status?id="_jovian.ssc"     can be run from command prompt with
**Start** http://localhost:8090/api/scripts/status?id="_jovian.ssc".
Both versions return {"runningScriptId":"_jovian.ssc","scriptIsRunning":**true**}, if the command is "fired" twice very fast, as expected.

**SET**
**curl** -d "id=_jovian.ssc" http://localhost:8090/api/scripts/run   the script is executed as expected, but I could not find the syntax of the
**Start** equivalence for this SET command...

I am fine with the **curl** version, but for making my scripts available in the future to a broader audience the **Start** version would be appreciated.
Can you help me, please?

Thank you

**Reply by gzotti** [reply]
No idea, sorry. I was not even aware of the `start URL`.

**Reply by 10110111** [reply]
Doesn't the default Windows' `start` command only launch the URL given via `ShellExecute` (so that, in particular, you see the result in a browser, rather than the console)? It's not a general purpose network tool. I wouldn't expect it to be able to do POST requests.

**Reply by gzotti** [reply]
Yes, that's also my understanding of the `start` command. It "opens" a document with the associated program.
